openpype/hosts/houdini/plugins/publish/collect_karma_rop.py

```diff
@@ -31,31 +32,33 @@ class CollectKarmaROPRenderProducts(pyblish.api.InstancePlugin):
     def process(self, instance):
 
         rop = hou.node(instance.data.get("instance_node"))
+        frame_start = instance.data["frameStart"]
+        frame_end = instance.data["frameEnd"]
 
         # Collect chunkSize
         chunk_size_parm = rop.parm("chunkSize")
         if chunk_size_parm:
             chunk_size = int(chunk_size_parm.eval())
             instance.data["chunkSize"] = chunk_size
-            self.log.debug("Chunk Size: %s" % chunk_size)
+            self.log.debug("Chunk Size: {}".format(chunk_size))
 
-            default_prefix = evalParmNoFrame(rop, "picture")
-            render_products = []
+        default_prefix = evalParmNoFrame(rop, "picture")
+        render_products = []
 
-            # Default beauty AOV
-            beauty_product = self.get_render_product_name(
-                prefix=default_prefix, suffix=None
-            )
-            render_products.append(beauty_product)
+        # Default beauty AOV
+        beauty_product = self.get_render_product_name(
+            prefix=default_prefix, suffix=None
+        )
+        render_products.append(beauty_product)
 
-            files_by_aov = {
-                "beauty": self.generate_expected_files(instance,
-                                                       beauty_product)
-            }
+        files_by_aov = {
+            "beauty": file_collection.generate_expected_filepaths(
+                frame_start, frame_end, beauty_product)
+        }
 
-            filenames = list(render_products)
-            instance.data["files"] = filenames
-            instance.data["renderProducts"] = colorspace.ARenderProduct()
+        filenames = list(render_products)
+        instance.data["files"] = filenames
```

Comment:
We don't seem to be using `filenames` anywhere else so I'd say: `instance.data["files"] = list(render_products)`